specs/DesignDiscussions/InputLabels.md

```diff
@@ -123,140 +115,130 @@ One option would be to make the label text a property of the input itself. This
 	]
 }
 ```
-This has the benefit of being simple and concise, but doesn’t immediately allow for formatting on the input label. A solution to that would be to add support for TextBlock and/or RichTextBlock labels to add formatting support. This is similar to the approach in Xaml:
 
-```json
-{
-	"type": "Input.Text",
-	"id": "nameInput",
-	"label": 
-	{
-		"type": "TextBlock",
-		"text": "Please enter your name:",
-		"weight": "Bolder",
-		"color": "Good"
-	},
-	"placeholder": "Last, First"
-}
-```
+## Formatting
+The above approach provides a simple means by which to specify the label, but doesn't immediately provide the ability to format the label. Card authors today who are using TextBlocks or RichTextBlocks as labels have the ability to set any of our formatting properties to control the size, color, font weight, etc. of their text. The schema proposed above does not give them that flexibility. One option to address this would be to allow card authors to specify a TextBlock or a RichTextBlock as a label in addition to supporting a simple string.
+
+There is another concern, however, which must be balanced against the desire to provide card authors with the ability to format their labels. Hosts may have label formatting in their app which they want to apply consistently to cards they are displaying. This applies particularly to how they mark fields as required or optional (per spec at [InputValidation.md](https://github.com/microsoft/AdaptiveCards/blob/master/specs/DesignDiscussions/InputValidation.md)). While by default we will mark required inputs with a `*`, we may want to allow hosts more control. Perhaps some apps wish to show labels for required inputs in bold or italics. If we allows hosts to specify formatting based on what is required, however, and we allow authors to specify formatting via a TextBlock or RichTextBlock, we can easily run into cases where the formatting comes into conflict.
+
+This conflict speaks to a general tension inherent in the design of Adaptive Cards. There's always going to be a trade off between giving the author more power, and letting the host control the experience for their users. We need to determine what balance we want to strike in this case.
 
-We should also consider which item types are valid as labels. Probably just TextBlock and RichTextBlock make sense, but as Xaml supports other types we should consider whether we want to as well.
+In order to gather more data to help us balance the needs of the card authors and hosts, we sent a survey to customers who have expressed an interest in improving out input story. The survey sent included 10 questions including the topics of input validation and input labels, for this spec we will only review the 4 questions that focus on the former topic. The results for this questions were used to leverage on some of the decisions taken on this spec document and can be found in the [Appendix](Appendix).
```

Comment:
>for this spec we will only review the 4 questions that focus on the former topic [](start = 277, length = 80)

Thanks for this detailed discussion of how the survey results were used! Great to see this feedback in action.